src/iadmin.cpp

```diff
@@ -1228,6 +1228,15 @@ doCommand( char *cmdToken[], rodsArguments_t* _rodsArgs = 0 ) {
     if ( strcmp( cmdToken[0], "moduser" ) == 0 ) {
         if ( strcmp( cmdToken[2], "password" ) == 0 ) {
             if (0 == strcmp(cmdToken[4], "no-scramble")) {
+                // If --no-scramble was specified, make sure the GeneralAdmin API of the connected server actually
+                // supports the feature. If not, exit with an error. The password will not be usable if it is set with
+                // this option and the server does not support this feature.
+                const auto version = irods::to_version(Conn->svrVersion->relVersion);
+                if (version && version.value() < irods::version{5, 0, 90}) {
+                    fmt::print("no-scramble option is not supported before iRODS 5.1.0.\n");
+                    return 0;
```

Comment:
Is there a way to indicate an error here?
Maybe there's an example of that in this function?

Reply:
Na, it's a known problem and can be handled in a separate PR.